src/sys/resource.rs

```diff
@@ -0,0 +1,69 @@
+use std::mem;
+
+use libc::{self, c_int};
+pub use libc::{rlimit, rlim_t, RLIM_INFINITY};
+
+#[cfg(any(target_os = "linux",
+          target_os = "openbsd",
+          target_os = "netbsd",
+          target_os = "bitrig"))]
```

Comment:
Bitrig no longer exists, do go ahead and drop this. But please add `fuchsia` and `empscripten`, as they also support this API.

Additionally these targets should be alphabetized.

Reply:
Done, removed the import.